pkg/schema/definitions/handler.go

```diff
@@ -0,0 +1,183 @@
+package definitions
+
+import (
+	"fmt"
+	"net/http"
+	"sync"
+	"time"
+
+	"github.com/rancher/apiserver/pkg/apierror"
+	"github.com/rancher/apiserver/pkg/types"
+	"github.com/rancher/steve/pkg/schema/converter"
+	"github.com/rancher/wrangler/pkg/schemas/validation"
+	"github.com/sirupsen/logrus"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/discovery"
+	"k8s.io/kube-openapi/pkg/util/proto"
+)
+
+var (
+	internalServerErrorCode = validation.ErrorCode{
+		Status: http.StatusInternalServerError,
+		Code:   "InternalServerError",
+	}
+	notRefreshedErrorCode = validation.ErrorCode{
+		Status: http.StatusServiceUnavailable,
+		Code:   "SchemasNotRefreshed",
+	}
+)
+
+type SchemaDefinitionHandler struct {
```

Comment:
nit: comment

Reply:
Added a comment on the struct and its internal fields.